src/hotspot/cpu/x86/gc/shenandoah/shenandoahBarrierSetAssembler_x86.cpp

```diff
@@ -140,10 +165,10 @@ void ShenandoahBarrierSetAssembler::arraycopy_prologue(MacroAssembler* masm, Dec
 #endif
       assert_different_registers(src, dst, count, thread);
 
-      Label done;
+      Label L_done;
```

Comment:
No need to rename the label? (This only clutters the patch and serves no purpose, IMO)

Reply:
This convention is used in many places in the barrier code:
```
% grep -riP 'Label.* L_done' src/hotspot | wc -l
44
```